docs/how-to/index.md

```diff
@@ -10,4 +10,5 @@ deploy_sdcore_cups
 deploy_sdcore_gnbsim
 integrate_sdcore_with_observability
 enable_hugepages_for_upf
```

Comment:
Not related to this PR specifically but in this file we tell the user to deploy the whole user plane bundle while in the `enable_hugepages_...` we only tell the user to deploy the upf charm. We should probably direct the user to install the bundle in both

Reply:
I agree. I was also wondering whether we should add the host configuration steps to the guide about enabling hugepages? 
Let me know what you think and I'll create JIRAs.

Reply:
I don't remember the rationale with the host config. Maybe @dariofaccin has more context. We can at least start with the bundle deployment (instead of the charm) and rename the "how-to" to be in line with the others (`deploy_sdcore_user_plane_with_hugepages`).

Reply:
For huge pages @gatici and I decided not to add specific commands because they may depend on the host. In particular it is necessary to edit the kernel command line parameters and some config files, and both these may change depending on the bootloader and/or OS.

Reply:
Yes, that's correct.

Reply:
That makes sense.
I've create [TELCO-901](https://warthogs.atlassian.net/browse/TELCO-901) to apply changes mentioned by @gruyaume